.github/workflows/cherryPick.yml

```
@@ -27,6 +24,22 @@ jobs:
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
           submodules: true
 
+      - name: Extract PR information
+        id: GetPRInfo
```

NAB we usually use camelCase for job ids:

```suggestion
        id: getPRInfo
```